### PR TITLE
Add syntax highlighting in Tutorial

### PR DESCRIPTION
**Pull request by dsh0416**
There've been some missing language declaration in Tutorial 3, caused missing syntax highlighting.
This pull request is to fix those problems.

### CODE REVIEW

File: TUTORIAL-3.md [hunk 1]
```diff
@@ -14,214 +14,236 @@ Suppose that we want our hello world app from tutorial 2 to insert a row into an
 
 Traveling Ruby provides a specific version of the sqlite3 gem. See [the Traveling Ruby Amazon S3 bucket](https://traveling-ruby.s3-us-west-2.amazonaws.com/list.html). For version 20141215-2.1.5, version 1.3.9 is supplied. So we add the following to our Gemfile:
 
-    gem 'sqlite3', '1.3.9'
+```Bash
```
Comment (FooBarWidget) [inline TUTORIAL-3.md]: This is supposed to be Ruby, not Bash.

File: TUTORIAL-3.md [hunk 2]
```diff
@@ -14,214 +14,236 @@ Suppose that we want our hello world app from tutorial 2 to insert a row into an
...
+db.execute("create table if not exists foo (name varchar(255))")
+db.execute("insert into foo values ('hello world')")
+db.close
+puts "Hello #{Faker::Name.name}, database file modified."
+```
 
 Then install your gem bundle:
 
-    $ bundle install
+```Bash
```
Comment (FooBarWidget) [inline TUTORIAL-3.md]: I don't think highlighting this is a good idea because the highlighter breaks at the prompt sign ($).

**Comment by FooBarWidget** [comment]
Thanks, I've added a few comments.

**Comment by dsh0416** [comment]
I've removed Bash declaration with '$'. Either removing Bash declaration or ($) symbol should be an acceptable solution.

**Pull request merged**